tests/e2e/map/map.spec.js

```diff
@@ -361,7 +361,7 @@ test.describe('Map interactions', () => {
       })
     })
 
-    test.describe.skip('When editing a circle', () => {})
+    test.describe.skip('When editing a circle', () => { })
```

Comment:
Added extra space

Reply:
Why are these conversations resolved if the extra space still exists?